src/NServiceBus.NHibernate/Outbox/OutboxPersister.cs

```diff
@@ -19,9 +20,9 @@ public bool TryGet(string messageId, out OutboxMessage message)
 
             message = null;
 
-            using (var session = StorageSessionProvider.OpenStatelessSession())
+            using (var session = SessionFactoryProvider.SessionFactory.OpenStatelessSession())
             {
-                using (var tx = session.BeginAmbientTransactionAware(IsolationLevel.ReadCommitted))
+                using (var tx = session.BeginTransaction(IsolationLevel.ReadCommitted))
```

Comment:
Is that always true? eg ASB calls it with a scope right now? (which is wrong but still)  Should we suppress to make sure?

or should the outbox behavior in the core suppress?